src/main/java/com/asana/Client.java

```diff
@@ -20,8 +21,7 @@
 /**
  * Main Asana client class
  */
-public class Client
-{
+public class Client {
```

Comment:
The repo is headed in this direction in terms of style. Will make a sweeping change in a different pull-request.